examples/main/package.json

```diff
@@ -13,11 +13,31 @@
     "immutable": "^3.8.1",
     "react": "^15.4.1",
     "react-dom": "^15.4.1",
-    "react-map-gl": ">=3.0.0-alpha.13"
+    "react-map-gl": ">=3.0.0-alpha.14"
   },
   "devDependencies": {
-    "buble-loader": "^0.4.0",
+    "autoprefixer-loader": "^3.2.0",
+    "babel-core": "^6.21.0",
+    "babel-loader": "^6.2.10",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
```

Comment:
Do we really want to use this in all our examples, we are already including `stage-2` to handle inline bound method definitions.